packages/analytics-js/src/components/core/Analytics.ts

```diff
@@ -288,20 +291,14 @@ class Analytics implements IAnalytics {
     this.storeManager?.init();
     this.userSessionManager?.init();
 
-    // Initialize consent manager
+    // Initialize the appropriate consent manager plugin
     if (state.consents.activeConsentManagerPluginName.value) {
-      this.pluginsManager?.invokeSingle(
-        `consentManager.init`,
-        state,
-        this.storeManager,
```

Comment:
storeManager not required anymore? in the plugins?

Reply:
It is not needed for initialization but only for another extension point (updateConsentsInfo) to fetch the user consents.

Reply:
Thanks for pointing it out anyway. I've fixed another bug.